practices/responsibility.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: responsibility
+stages:
+  - development
+short_description: Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is. Responsibility primarily involves two things -- authority and accountability; however, trust is often a factor as well.
+
+tags:
+  - responsibility
+  - team work
+  - ability
+  - response
+  - appreciation
+  - problems
+  - problem-solving
+  - assign
+  - leader
+
+keywords:
+  - responsibility
+  - trust
+  - people
+  - benefits
+---
+
+# Responsibility
+
+**TL;DR**
+
+Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is.
+
+
+## What Is Responsibility
+Responsibility is the combination of two words: **Response + ability**. It also means taking on more risk and therefore reward because it involves a degree of success and failure beyond oneself. People are responsible for themselves and so when failure happens, you are just failing yourself. When you choose to take on responsibilities beyond yourself, you are agreeing to take the interests of others into your hands in trying to do more, be more, and allow others to benefit from your being responsible.
+
+Responsibility is not for everyone. But it’s a powerful and deeply rewarding ship to navigate once embarked.
+
+![Responsibility](https://shiftyourfamilybusiness.com/wp-content/uploads/2018/05/ResponsibilityBlog-845x321.jpg)
+
+
+## Why You Might Want Responsibility
+
+Being responsible brings us many benefits. It can help us achieve our goals and objectives in any area of our life. Responsibility allows us to create principles, morals and helps us to lead your life. Being a responsible person helps us to:
```

Comment:
This part has similarities with _Becoming a responsible person means being able to consciously make decisions, conduct behaviors that seek to improve oneself and/or help others. Most importantly, a responsible person accepts the consequences of his or her own actions and decisions._ from https://blog.cognifit.com/responsibility/